### ISSUE TITLE
Feature - Stop SteamVr restarting

### ISSUE THREAD

**Issue by sutex** [TuTAH1/SteamVR-Toggle-for-Viveport#7]
Kill steamvr - shuts down, closes, Oculus Home first, otherwise SteamVr restarts

I'm running Oculus Tray Tool, and checked. " Stop Oculus service when Oculus Home is closed"

**Comment by TuTAH1** [comment]
I don't quite understand what do you mean. Are you about SteamVR restarts on close if SteamVR mode is choosen?

**Comment by sutex** [comment]
Yes, when running the Oculus Rift, in SteamVR , and using kill steamVr. SteamVR was restarting

**Comment by TuTAH1** [comment]
> Yes, when running the Oculus Rift, in SteamVR , and using kill steamVr. SteamVR was restarting

Well, I'm not sure I can fix that, becouse it's seems OculusDashKiller launching SteamVR, but I can add an option to Break SteamVR after killing it, so it will be unable to be launched again.

For now, you can enable some SteamVR-breaking method in settings and use "SteamVR/Oculus" switch to kill&break/restore SteamVR. Or just switch to oculus mode, for me it works

**Comment by TuTAH1** [comment]
I'll check if I can stop SteamVR restarting on "Kill SteamVR" click by also killing OculusDash.exe if "Kill Oculus Dash" option active
